piwik.php

```diff
@@ -182,7 +241,7 @@ function getHttpContentAndStatus($url, $timeout, $user_agent)
 
     return array(
         $content,
-        $httpStatus
+        $httpStatus,
```

Comment:
there is a trailing comma, but not important

Reply:
Left when undoing a change. Will leave it for the better blame.